tools/CMakeLists.txt

```diff
@@ -200,6 +200,29 @@ if(TARGET pcl_filters)
     endif()
   endif()
 
+  if(TARGET pcl_outofcore)
+    # pcl_outofcore_process
+    PCL_ADD_EXECUTABLE(pcl_outofcore_process COMPONENT ${SUBSYS_NAME} SOURCES outofcore_process.cpp)
+    target_link_libraries(pcl_outofcore_process pcl_common pcl_io pcl_octree pcl_filters pcl_outofcore)
+
+    PCL_ADD_EXECUTABLE(pcl_outofcore_print COMPONENT ${SUBSYS_NAME} SOURCES outofcore_print.cpp)
+    target_link_libraries(pcl_outofcore_print pcl_common pcl_io pcl_octree pcl_filters pcl_outofcore)
+
+    if(TARGET pcl_visualization)
+      set(srcs outofcore_viewer.cpp
+        ../outofcore/src/visualization/camera.cpp
```

Comment:
I don't like the idea of using/referring to files in another PCL module. However, it seems like the files in outofcore/src/visualization are only used by the outofcore_viewer, and not compiled into the outofcore module library? But on the other hand, the corresponding headers in outofcore/include/pcl/outofcore/visualization are installed as if the outofcore module library contained all the stuff from the cpp files? Unfortunately I don't have any experience with outofcore, so no idea how it is supposed to be.
Moving pcl_outofcore_process and pcl_outofcore_print seems unproblematic, so that's fine by me